modules/navigation/nav_map.h

```diff
@@ -136,6 +136,13 @@ class NavMap : public NavRid {
 	HashMap<gd::EdgeKey, ConnectionPair, gd::EdgeKey> connection_pairs_map;
 	LocalVector<gd::Edge::Connection> free_edges;
 
+	struct {
+		SelfList<NavRegion>::List region_list;
+		SelfList<NavLink>::List link_list;
+		SelfList<NavAgent>::List agent_list;
+		SelfList<NavObstacle>::List obstacle_list;
+	} sync_dirty_requests;
```

Comment:
wouldn't something like `dirty_objects_to_sync`, `objects_to_sync`, `entities_to_sync` or something like that be more natural? `sync_dirty_requests` sounds very artificial

Reply:
In my branch I have other sync points as well, e.g. for iterations, so I wanted them to follow a sync_xyz pattern that somehow also matches with the functions. I agree it sounds artificial although I have no better idea atm.

Reply:
ok, then it's good enough for now